src/Compilers/CSharp/Portable/Syntax/SyntaxKindFacts.cs

```diff
@@ -204,6 +204,12 @@ public static bool IsAnyToken(SyntaxKind kind)
                 case SyntaxKind.InterpolatedVerbatimStringStartToken:
                 case SyntaxKind.InterpolatedStringTextToken:
                 case SyntaxKind.InterpolatedStringEndToken:
+                case SyntaxKind.RawInterpolationCloseToken:
+                case SyntaxKind.RawInterpolationOpenToken:
+                case SyntaxKind.MultiLineRawInterpolatedStringEndToken:
+                case SyntaxKind.MultiLineRawInterpolatedStringStartToken:
+                case SyntaxKind.SingleLineRawInterpolatedStringEndToken:
+                case SyntaxKind.SingleLineRawInterpolatedStringStartToken:
```

Comment:
we explicitly don't need to update GetText for these kinds because they can all be variable width.